src/vl-pattern.js

```diff
@@ -7,8 +7,9 @@ import '/lib/pattern.js';
  * Gebruik de pattern mixin in combinatie met een input field om de gebruiker te verplichten om informatie in een bepaald formaat op te geven.
  * @mixin vlPattern
  *
- * @property {(iban | phone | date | price | rrn | uuid)} data-vl-pattern - Attribuut wordt gebruikt om aan te duiden welk patroon van toepassing is.
+ * @property {(iban | phone | date | price | rrn | uuid | numerical)} data-vl-pattern - Attribuut wordt gebruikt om aan te duiden welk patroon van toepassing is.
  * @property {string} data-vl-pattern-prefix - Attribuut bepaalt de prefix die de gebruiker zal zien. In het geval van iban en phone pattern type wordt de prefix toegevoegd aan de value. Bij price pattern type is de prefix puur visueel.
+ * @property {number} [2] data-vl-numerical-decimal-scale - Attribuut bepaalt hoeveel getallen er achter de komma ingevoerd kunnen worden.
```

Comment:
misschien zeggen dat het enkel bij pattern numerical van belang is?

Reply:
In principe kan het bij sommige anderen ook gebruikt worden, zoals price. 
Ik heb niet gekeken bij wat allemaal, dus wilde dat liever niet "beperken" via de documentatie.
Maar ja, als meer mensen dit vinden pas ik het wel aan.